docs/user_guide.md

```diff
@@ -751,29 +751,46 @@ then the results of the analysis can be printed with
 CodeChecker parse ./my_plists
 ~~~~
 
-### <a name="suppression-code"></a> Suppression in the source code
+### <a name="suppression-code"></a> Source code comments in the source code
 
-Suppress comments can be used in the source files to suppress specific or all
-checker results found in a particular line of code. Suppress comment should be
-above the line where the defect was found, and no empty lines are allowed
-between the line with the bug and the suppress comment.
+Source code comments can be used in the source files to suppress specific or all
```

Comment:
@gyorb has a comment here still unfixed.

Reply:
@whisperity I've fixed that. Check it below (https://github.com/csordasmarton/codechecker/blob/9ecdb6b633377dcfdb94994294cf77eb1d60e678/docs/user_guide.md#-source-code-comments-in-the-source-code).

Reply:
Uh-huh. But how does this... I mean I think saying "to suppress" isn't a good wording here, because... the analyzer report will still be visible, just with a different review status.

IIRC, originally if something was suppressed, they were put into a much different category on the GUI and the user had to *explicitly* ask for showing suppressed reports, same with command-line.

That's why I think the first sentence should be rephrased to talk more about review status and less about suppression.